app/src/main/java/org/schabi/newpipe/local/subscription/item/FeedImportExportItem.kt

```diff
@@ -115,7 +115,7 @@ class FeedImportExportItem(
     private fun setupExportToItems(listHolder: ViewGroup) {
         val previousBackupItem = addItemView(
             listHolder.context.getString(R.string.file),
-            ThemeHelper.resolveResourceIdFromAttr(listHolder.context, R.attr.ic_save),
+            R.drawable.ic_save,
             listHolder
```

Comment:
Consistency with above
```suggestion
            R.drawable.ic_save, listHolder
```